apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientImpl.cs

```diff
@@ -62,6 +62,15 @@ internal SubscriberClientImpl(SubscriptionName subscriptionName, IEnumerable<Sub
         _modifyDeadlineSeconds = (int)((settings.AckDeadline ?? DefaultAckDeadline).TotalSeconds);
```

Comment:
I'm wondering how big of a change would it be to rename _modifyDeadlineSeconds to _ackDeadlineSeconds.
and honestly a lot of the other names here are somewhat confusing, for instance autoExtendInterval would be clearer if it was called delayBeforeAutoExtend (as it is really the delay before )
Obviously if we were to make some of those changes, they would be on their own commits or even on a separate PR, but I think renaming some things here would help a lot.
@jskeet for thoughts.

Reply:
Renamed them as _ackDeadlineSeconds and _autoExtendDelay.